a10_octavia/common/config_options.py

```diff
@@ -63,9 +63,6 @@
     cfg.IntOpt('default_axapi_timeout',
                default=300,
                help=_('VThunder axapi timeout')),
-    cfg.BoolOpt('l2dsr_support', default=False,
```

Comment:
I think we still need this option for backward compatibility.

Reply:
Done